### PR TITLE
tabling people on brass tables makes a clanging sound

### PR DESCRIPTION
**Pull request by Buggy123**
@kevinz000

### CODE REVIEW

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -338,7 +338,10 @@
 /obj/structure/table/reinforced/brass/Destroy()
 	change_construction_value(-2)
 	return ..()
-
+	
+/obj/structure/table/reinforced/brass/tablepush(mob/living/user, mob/living/pushed_mob)
+	..()
```
Comment (Cyberboss) [inline code/game/objects/structures/tables_racks.dm]: . = ..()

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/tables_racks.dm [hunk 2]
```diff
@@ -338,7 +338,10 @@
 /obj/structure/table/reinforced/brass/Destroy()
 	change_construction_value(-2)
 	return ..()
-
+	
+/obj/structure/table/reinforced/brass/tablepush(mob/living/user, mob/living/pushed_mob)
+	.
```
Comment (ShizCalev) [inline code/game/objects/structures/tables_racks.dm]: = ..()

**Review by ShizCalev — CHANGES_REQUESTED** [review]
still missing

**Comment by Buggy123** [comment]
w e w

**Pull request merged**